### ISSUE TITLE
Output filter field icon cannot be tabbed to, only shift-tabbed

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#117354]
Testing #116559

![recording (1)](https://user-images.githubusercontent.com/900690/108829114-10298e80-75c8-11eb-9e37-9ef428f08fd6.gif)

**Comment by isidorn** [comment]
Yeah that is a corner case. Will not fix in endgame. Will investigate in debt week.

**Comment by jeanp413** [comment]
Also found that when the filter field is positioned inside the pane body when the panel is narrow (like when moving the panel to the side) you cannot continue tabbing
![output_small_filter_tab](https://user-images.githubusercontent.com/25115070/109393116-08caf380-78ee-11eb-9e3a-7904a68cfb8f.gif)

**Comment by isidorn** [comment]
@jeanp413 Thanks for the PR I will review it later today or tomorrow.
As for the smaller panel trapping tab can you open a new issue for this and then we can investigate? Thanks for catching that

**Comment by jeanp413** [comment]
I fixed it in the same PR

**Comment by isidorn** [comment]
Great, thanks

**Comment by isidorn** [comment]
@jeanp413 thanks a lot for fixing 👏
